source/_integrations/music_assistant.markdown

```diff
@@ -183,19 +203,91 @@ Perform a local search on the Music Assistant library.
   - **Description**: When `album` is the `media_type` then this option will restrict the result according to the selection of either album, single, compilation, EP or unknown.
   - **Example**: `album`
 
+#### Example
+
+This example will start playback of ten random tracks.
+
+```yaml
+script:
+  create_random_queue:
+    mode: single
+    sequence:
+      - service: music_assistant.get_library
+        data:
+          limit: 10
+          media_type: track
+          config_entry_id: 01JEXNDHT21V0BHJXM7A5SZANV
+          order_by: random
+        response_variable: random_tracks
+      - repeat:
+          count: "{{ random_tracks['items'] | length }}"
+          sequence:
+            - action: music_assistant.play_media
+              data:
+                media_id: "{{ random_tracks['items'][repeat.index-1].uri }}"
+                media_type: track
+                enqueue: add
+              target:
+                entity_id: media_player.ma_kitchen_speaker
```

Comment:
Why not simply assign the result in a new array of just uri's ?

Reply:
I could. I have used that somewhere myself.